src/lqcut.jl

```diff
@@ -1,60 +1,76 @@
 """
-    lq_drift_time_correction(lq_norm::Vector{Float64}, tdrift, e_cal; DEP_left=1589u"keV", det_icpc = false, DEP_right=1596u"keV", lower_exclusion=0.005, upper_exclusion=0.98, drift_cutoff_sgima=2.0)
+    lq_norm::Vector{Float64}, dt_eff::Vector{<:Unitful.RealOrRealQuantity}, e_cal::Vector{<:Unitful.Energy{<:Real}}, DEP_µ::Unitful.AbstractQuantity, DEP_σ::Unitful.AbstractQuantity; 
```

Comment:
Is there a reason why we require here `Float64`? I find this a bit strict. Let's say at some point we wanna store DSP output as `Float32` to save some diskspace, all the routines would fail.
Let's change that.